Update presentation (#150)
alsuren/mijia-homie: docs/presentation/slides.md

# Bluetooth

The Rust Bluetooth story is a bit sad.

- `blurz` - "Bluetooth from before there was Tokio"
  - Started with this.
  - Talks to BlueZ over D-Bus, but single-threaded and synchronous.
  - Blocking `device.connect()` calls. 😧
  - Unmaintained (for 2 years).

<!-- prettier-ignore-start -->

- `btleplug` - "cross-platform jumble"
  - Theoretically cross platform, but many features not implemented.
  - Linux implementation talked to kernel directly over raw sockets, bypassing BlueZ daemon.
      - Requires extra permissions, adds extra bugs.
      - This has since been changed.
  - Tried switching to this (but gave up after too many panicking threads).

>   - Andrew now trying to fix it and make it async.

Reviewer: ```suggestion
  - Andrew now working to make it suit our async.
```

Author: Done.

> - Andrew is good at separating things into modules (and crates):

Reviewer: ```suggestion
- Separating things into modules (and crates) worked well:
```

Author: Done.
